src/azure-cli-core/azure/cli/core/aaz/_arg_action.py

```diff
@@ -58,7 +58,9 @@ class AAZArgAction(Action):
     _str_parser = AAZShortHandSyntaxParser()  # support to parse normal string or short hand syntax string
 
     def __call__(self, parser, namespace, values, option_string=None):
-        if getattr(namespace, self.dest) is None or getattr(namespace, self.dest) == AAZUndefined:
+        if not isinstance(getattr(namespace, self.dest), AAZArgActionOperations):
+            # overwrite existing namespace value which is not an instance of AAZArgActionOperations.
+            # especially the default value of argument.
             setattr(namespace, self.dest, AAZArgActionOperations())
         dest_ops = getattr(namespace, self.dest)
```

Comment:
When an argument has a default vaule, it will be assigned to namespace beforehand. In that case, the value of it is not an instance of AAZArgActionOperations, which will cause issues in the following code.